setup.sh

```diff
@@ -74,3 +74,27 @@ if [[ ${PROJECT_TYPE} == 'nodejs' ]]; then
   echo "install npm modules ${NPM_MODULES[@]}"
   npm i ${NPM_MODULES[@]} --save-dev
 fi
+
+read -p "Use default folder structure? [y, N]: " DEFAULT_FOLDER_STRUCTURE </dev/tty
+
+if [[ ${DEFAULT_FOLDER_STRUCTURE} == 'y' ]]; then
+
+  mkdir -p app/{jade/layouts,styles,scripts}
+
+  echo "Created folder structure:"
+  find app -type d -print
+
+  fetch "jade-default"
+  mv jade-default app/jade/layouts/_default.jade
```

Comment:
Please use files names without underscores. This concept is old. Per default we compile targets from teh root folder and the layout should be private at all circumstances.